src/lib/components/admin/configuration/cell/PrivilegeActions.svelte

```diff
@@ -19,7 +20,7 @@
       buttonTextConfirm: 'Yes',
       buttonTextCancel: 'No',
       response: async (confirm: boolean) => {
-        if (!confirm) return;
+        if (!confirm || !$isTopAdmin) return;
 
```

Comment:
Maybe just `if(!$isTopAdmin) return;` at the top of this (and similar) method would be better. Probably doesn't matter that much though.

Reply:
tbh I dont know why I put it there, they should never get this far